ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/RefragmentationServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.entities.LdesFragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.entities.Member;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.repository.MemberRepository;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import org.springframework.stereotype.Service;
+
+import java.util.stream.Stream;
+
+@Service
+public class RefragmentationServiceImpl implements RefragmentationService {
+	private final MemberRepository memberRepository;
+	private final ObservationRegistry observationRegistry;
+
+	public RefragmentationServiceImpl(MemberRepository memberRepository, ObservationRegistry observationRegistry) {
+		this.memberRepository = memberRepository;
+		this.observationRegistry = observationRegistry;
+	}
+
+	// TODO refragmentation of old members might take a while. During this period
+	// new arrived members will not be fragmented.
+	// In the server v2 (horizontal scaling) we should solve this. For example by
```

Comment:
Isn't this server v2?

Reply:
Yes, but we won't be able to do horizontal scaling and the dynamically updating of config in one go. Hence, the remark that in this PR. We handle the dynamically updates of a view.

(see also the next remark)